src/compute/src/server.rs

```diff
@@ -167,21 +168,21 @@ pub async fn compute_node_serve(
     let mut sub_tasks: Vec<(JoinHandle<()>, Sender<()>)> = vec![];
     // Initialize the metrics subsystem.
 
-    monitor_process(&registry).unwrap();
-    let source_metrics = Arc::new(SourceMetrics::new(registry.clone()));
-    let hummock_metrics = Arc::new(HummockMetrics::new(registry.clone()));
-    let streaming_metrics = Arc::new(StreamingMetrics::new(registry.clone()));
-    let batch_task_metrics = Arc::new(BatchTaskMetrics::new(registry.clone()));
-    let batch_executor_metrics = Arc::new(BatchExecutorMetrics::new(registry.clone()));
-    let batch_manager_metrics = BatchManagerMetrics::new(registry.clone());
-    let exchange_srv_metrics = Arc::new(ExchangeServiceMetrics::new(registry.clone()));
+    monitor_process();
```

Comment:
What about launching this directly in the initialization of `GLOBAL_REGISTRY`?

Reply:
Good idea! I have changed to calling it when initializing global registry.